common.mk

```diff
@@ -159,9 +177,14 @@ $(FIRTOOL_TARGETS) &: $(FIRRTL_FILE) $(FINAL_ANNO_FILE) $(VLOG_SOURCES)
 		--annotation-file $(FINAL_ANNO_FILE) \
 		--log-level $(FIRRTL_LOGLEVEL) \
 		--allow-unrecognized-annotations \
-		-X $(if $(shell grep "Fixed<" $(FIRRTL_FILE)),middle,none) \
+	  $(REPL_SEQ_MEM) \
+		$(HARNESS_CONF_FLAGS) \
+		-X $(SFC_LEVEL) \
 		$(EXTRA_FIRRTL_OPTIONS))
-	$(if $(shell grep "Fixed<" $(FIRRTL_FILE)),mv $(SFC_FIRRTL_BASENAME).mid.fir $(SFC_FIRRTL_FILE),)
+	-mv $(SFC_FIRRTL_BASENAME).mid.fir $(SFC_FIRRTL_FILE)
+	-mv $(SFC_FIRRTL_BASENAME).lo.fir $(SFC_FIRRTL_FILE)
+	$(if $(ENABLE_CUSTOM_FIRRTL_PASS), cat $(SFC_ANNO_FILE) | jq 'del(.[] | select(.target | test("io.cpu"))?)' > $(build_dir)/tmp.json,)
```

Comment:
I thought you just needed to delete the `SRAMAnnotation`? What is this `io.cpu`? and `select...` stuff?

Reply:
Also can we not do this within the same file? If not, maybe dump the tmp file to `/tmp/descriptive-name-here`.

Reply:
By not in the same file, do you mean make a script that does this `jq` stuff? After adding back the `REPL_SEQ_MEM` option, all the `SRAMAnnotation` stuff went away & only the `io.cpu.dcache` related things (they were all `dontTouchTransforms` & they were there with the `SRAMAnnotation`s when transforming to LowFIRRTL).

Reply:
Basically is there a way to have `jq` operate in place. In other words, instead of generating a `tmp.json` file, can you have a single `jq` operation that does `jq --some-args --in-place $(JSON)` (like `sed -e`)

Reply:
I had to use `tmp.json` because if I did something like `cat some_file | jq 'some operation' > some_file` (which would be the commands w/o `tmp.json`), `some_file` became a blank file. I'll update it to use `/tmp/descriptive-name`.

Reply:
`jq` supports this, but for some reason, if I do something like `jq do-sth input_file > input_file`, the `input_file` becomes blank.

Reply:
Can you add a comment on the `io.cpu` `DontTouchAnnos` causing issues and needing to be deleted from the JSON. I can see this being another issue in the future if people put `DontTouchAnnos` on SRAM items.